specification/oep/resource-manager/Microsoft.OpenEnergyPlatform/preview/2021-06-01-preview/oep.json

```diff
@@ -0,0 +1,517 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Open Energy Platform Management Service APIs",
+    "description": "Open Energy Platform Management Service APIs",
+    "version": "2021-06-01-preview"
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.OpenEnergyPlatform/checkNameAvailability": {
+      "post": {
+        "tags": [
+          "Locations"
+        ],
+        "description": "Checks the name availability of the resource with requested resource name.",
+        "operationId": "Locations_CheckNameAvailability",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "NameAvailabilityRequest object.",
+            "required": true,
+            "schema": {
+              "description": "NameAvailabilityRequest object.",
+              "$ref": "#/definitions/CheckNameAvailabilityRequest"
```

Comment:
that break the consistency we have with azure-rest-api-specs-pr repo. Is that acceptable?

Reply:
I do not think this will break the consistency since you are already using `SubscriptionIdParameter` from common-types. Could we do the same for this request?
If you cannot, please make sure we have exactly the same definition in this swagger with the one in common-types (we should align the descriptions for instance)